wg-meetings/2018-01-10.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Diagnostics Working Group Meeting 2018-01-10 Minutes
+
+
+## Attendees
+
+- @mike-kaufman
+- @qard(Stephen Belanger)
+- @AndreasMadsen
+- @danielkhan
+- @jankrems
+- @kjin
+- @ofrobots
+- @dominicdkramer
+- @mhdawson
+
+## Links
+
+* **This Meeting**: 
+  * Minutes:  https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2018-01-10.md
+  * Notes: https://docs.google.com/document/d/1Zb92YTZQWf8i73UrRfusID7zfLtAcjYvmqUBDbORsOE/edit
+  * You Tube: N/A
+* **Previous Meeting**:
+  * Minutes:  https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2017-12-13.md
+  * Notes: https://docs.google.com/document/d/1ZkTGBUk5lgodtwtlY_0eoInpctVGpf6ofB5TyUF0hqs/edit?usp=sharing
+
+## Notes
+
+### nodejs/node
+
+* Integrate C++ AsyncHooks Embedder API with native abstraction [#13254](https://github.com/nodejs/node/issues/13254)
+  - issue is stalled
+  - how do we communicate this with a broader group?
+  - **Actions**
+    - [ ] @ofrobots volunteered to look into this
+
+### nodejs/diagnostics
+
+* \[async-hooks\] request for feedback [#129](https://github.com/nodejs/diagnostics/issues/129)
+  - Do we have enough feedback & is it useful? 
+  - [bluebird issue #1472](https://github.com/petkaantonov/bluebird/pull/1472) on perforance of async hooks is related to this.
+  - @danielkhan -  they're support down to node v4, and haven't had time to invest in a 8.x/9.x solution only - their monkey-patching still works
+  - @ofrobots - want to know if API is stable & useful
+  - **Actions**
+    - [ ] @mike-kaufman - summarize feedback in here & roll it into [async_hooks] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124) 
+
+* \[async_hooks\] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124)
+  - 

```

Comment:
Github interprets this trailing dash as a header:

![image](https://user-images.githubusercontent.com/4048656/34846703-2b424d6e-f700-11e7-9f4f-a198be366b6a.png)

Reply:
fixed